CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 Use the following format for additions: ` - VERSION: [feature/patch (if applicable)] Short description of change. Links to relevant issues/PRs.`
 
+- 1.2.0:
+    - Fix cli arguments [#871](https://github.com/FredrikNoren/ungit/pull/871)
+    - Ignore ~/.ungitrc if contains syntax error
+    - Removed official support ini format of ~/.ungitrc, because internal API supports only JSON
```

Comment:
Ah, can we do 1.1.10?  I would like to reserve minor version bump to when stability is confirmed and code is more mature after code freeze.  I'd imagined @FredrikNoren wants that.